analysis/backtrace/backtrace.go

```diff
@@ -845,6 +864,27 @@ func findTrace(s *df.AnalyzerState, end *df.VisitorNode) Trace {
 	return trace
 }
 
+func addTrace(v *Visitor, entrypoint *df.CallNodeArg, trace Trace) {
+	// don't add trace if it's already in v.Traces
+	for _, t := range v.Traces[entrypoint] {
+		// need this more complex logic because you can't compare slices directly
```

Comment:
There is theoretically reflect.DeepEqual() that could do this in one line, but I'm not sure that's any better